packages/ra-ui-materialui/src/list/filter/FilterButton.stories.tsx

```diff
@@ -372,3 +383,65 @@ export const WithAutoCompleteArrayInput = (args: {
         </MemoryRouter>
     );
 };
+
+export const WithButtonVariant = () => {
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏May we name it as the prop name

```suggestion
export const Variant = () => {
```